dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
>I think writing our own tools is fine, but make them as generic as possible.

If there are tools around that do the same and have proven stable, why not use them?
I am just saying that we *might* introduce unwanted issues which have an impact on the security (e.g. see https://github.com/RIOT-OS/RIOT/pull/6450/files#r97352140). Other established tools might also adapt more quickly to changes in the random internals of the host OS, while we would actively need to maintain the (in this PR) proposed toolset.